feat: add support for sending messages in supergroups (#66)
cbrgm/telegram-github-action: README.md

- `token`: **Required** - Telegram bot's authorization token. Use GitHub secrets.
- `to`: **Required** - Unique identifier or username of the target Telegram chat.

> - `thread-id`: Optional - Identifier of a topic in a Telegram supergroup.
>     - The message would not be sent if `to` is not a supergroup and `thread-id` is set.

Reviewer: ```suggestion
- `thread-id`: (Optional) Topic identifier in a Telegram supergroup.  
    - The message won’t be sent if `to` isn’t a supergroup and `thread-id` is set.
```

Reviewer: Same here, is topic different from thread? We should stay consistent with the naming to not cause confusion.

```
Replace <token> with your bot's token to find your `chat_id` (`TELEGRAM_CHAT_ID`).

> ### Obtaining the Topic ID
> To get the topic ID, the easiest way is this: Post a message to that topic, then right-click on it and select Copy Message Link . Paste it on a scratchpad and notice that it has the following structure https://t.me/c/XXXXXXXXXX/YY/ZZ . The topic ID is YY (integer).
> 

Reviewer: ```suggestion
### Obtaining the Topic ID
The easiest way to get the topic ID is as follows: Post a message in that topic, then right-click it and choose Copy Message Link. Paste it onto a scratchpad and you will notice that it has the following structure https://t.me/c/XXXXXXXXXX/YY/ZZ . The topic ID is YY (integer).
```

Also is there a reason why it's called topic ID here. Is topic ID different from thread ID?
